src/community/ogcapi/ogcapi-core/src/main/java/org/geoserver/ogcapi/ConformanceInfo.java

```diff
@@ -0,0 +1,91 @@
+package org.geoserver.ogcapi;
+
+import org.geoserver.config.ServiceInfo;
+
+/**
+ * Configuration for a service capability or feature, identified by conformance class.
```

Comment:
I'd clarify that this is indicate the presence and configuration of a given conformance class in a specific service. E.g., one will have the sortby conformance class in both STAC, Features and Records.

The HTML and JSON conformance classes are likely going to be found everywhere, but it's not something we likely want to expose as configurable. Same goes for the various Part 1, they are there because the service is there, if one does not want it, the service is not enabled (that's however normally controlled by the enabled flag in the ServiceInfo)

Reply:
I need to mark some of the APIConformance as *required*, not subject to enabled/disabled. It would still be nice to list them. 

Somehow I need to turn these things into beans, and I am not quite sure where to begin.